src/main/java/com/google/sps/data/User.java

```diff
@@ -5,28 +5,54 @@
 
 public final class User {
 
+  private final long id;
+  private final String profileType;
   private final String name;
-  private final Collection<String> productArea;
+  private final String school;
+  private final String major;
+  private final String company;
+  private final String occupation;
+  private final String speciality;
+  private final Collection<String> specialties;
 
   public User(String name) {
```

Comment:
I would suggest having a single constructor for mandatory and then using setter methods for optional fields (e.g. Specialty). Maintaining multiple constructors may require tests for each one which may be harder to maintain. 

Thoughts? @samuelsandoval1 @hadleyshapland

Reply:
I agree ok